pkg/pxc/backup/restore.go

```diff
@@ -263,6 +287,25 @@ func RestoreJob(cr *api.PerconaXtraDBClusterRestore, bcp *api.PerconaXtraDBClust
 		return nil, errors.Wrap(err, "restore job envs")
 	}
 
+	if cluster.CompareVersionWith("1.17.0") >= 0 && !pitr {
```

Comment:
Why are we asserting with 1.17 and not 1.18?

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/2065/commits/9640cdc360c22aed87f34c7cf1ddb5ba5dc5fdd1